package.json

```diff
@@ -3,51 +3,55 @@
   "version": "0.1.89-SNAPSHOT",
   "private": true,
   "dependencies": {
-    "clarinet": "0.11.0",
     "async": "1.5.2",
-    "babel": "~5.8.29",
-    "body-parser": "1.14.2",
-    "bunyan": "1.5.1",
+    "babel": "^5.8.38",
```

Comment:
Pretty much every dependency that has a `^` will need to have that removed, there might also be other issues we will need to check. (I just ran into one recently)